superduperdb/ext/llm/__init__.py

```diff
@@ -0,0 +1,9 @@
+from superduperdb.ext.llm.openai import OpenAI
+from superduperdb.ext.llm.vllm import VllmAPI, VllmModel, VllmOpenAI
+
+__all__ = [
```

Comment:
A general question: how does this fit together with the other `OpenAI` models which are not of the chat-completion type?

Reply:
A `chat` parameter handles this. If set `chat=False` then it will call _prompt_generate(`self.client.completions.create`), otherwise it will call _chat_generate(`self.client.chat.completions.create`)

Reply:
No I mean we have `superduperdb.ext.openai.*`; what do we do with those models?

Reply:
Oops got it, we should continue to maintain these modules as support for OpenAI vendors